lib/registrant.rb

```diff
@@ -0,0 +1,54 @@
+require './lib/vehicle'
+
+class Registrant
+    attr_reader :name, 
+                :age, 
+                :permit,
+                :license_data,
+                :vehicles,
+                :year
+    attr_accessor :balance
+    def initialize(name, age, permit = false)
+        @name = name 
+        @age = age
+        @permit = permit
+        @year = year
+        @license_data = {:written=>false, :license=>false, :renewed=>false}
+        @vehicles = []
+        @balance = 0
+        @antique = antique?
```

Comment:
This line is causing your tests to fail, as the stack trace indicates when I run `bundle exec rspec`:
```
  1) Facility#initialize can initialize
     Failure/Error: @antique = antique?
     
     NoMethodError:
       undefined method `antique?' for #<Registrant:0x000000010fbc2158 @name="Bruce", @age=18, @permit=true, @year=nil, @license_data={:written=>false, :license=>false, :renewed=>false}, @vehicles=[], @balance=0>
     
               @antique = antique?
                          ^^^^^^^^
     # ./lib/registrant.rb:19:in `initialize'
     # ./spec/facility_spec.rb:7:in `new'
     # ./spec/facility_spec.rb:7:in `block (2 levels) in <top (required)>'
```
I think the important takeaway here is that a `Registrant` should not have any concept of `antique?` because that is an attribute of a `Vehicle`. The `antique?` method that does live in the Vehicle class looks good to me-- we should remove this one.